roles/telemetry_autoscaling/tasks/post_teardown.yml

```diff
@@ -14,10 +14,9 @@
 
 - name: Wait until the stack is deleted
```

Comment:
I don't think this ever worked correctly. In a situation when this should pass, the original output.stdout_lines would be an empty list, not 0. I checked a recent CI run and this actually failed. I added | wc -l so the check is now similar to a few other ones from this role.

Reply:
This should never have worked. I suspect that the ``ignore_errors: true`` is also not needed, and may have been the reason that this wasn't causing issues